### DISCUSSION TITLE
Dec Newsletter - Cloudberry Database (2023/12)

### DISCUSSION THREAD

**Discussion by tuhaihe** [apache/cloudberry#353] [Announcements]
Hello there! We hope you're having a great time during the holiday season. Thank you for joining us for the latest Cloudberry Database project newsletter in December!

In this newsletter, we will be sharing the latest updates, improvements, and contributions made by our incredible members. Let's dive right in and explore what's new!

## Pull Requests

- #352 by @songdongxiaoa2
- #351 by @gfphoenix78
- #350 by @shmiwy
- #349 by @gfphoenix78
- #348 by @avamingli
- #347 by @gfphoenix78
- #346 by @gfphoenix78
- #343 by @gfphoenix78
- #342 by @avamingli
- #341 by @gfphoenix78
- #340 by @gfphoenix78
- #339 by @Ray-Eldath
- #338 by @gfphoenix78
- #337 by @gfphoenix78
- #336 by @gfphoenix78
- #335 by @gfphoenix78
- #333 by @gfphoenix78
- #332 by @gfphoenix78
- #331 by @gfphoenix78
- #330 by @gfphoenix78
- #329 by @gfphoenix78
- #328 by @gfphoenix78
- #327 by @avamingli
- #326 by @jiaqizho
- #325 by @jiaqizho
- #324 by @tuhaihe
- #322 by @avamingli
- #321 by @avamingli
- #320 by @oppenheimer01

## Issues

- #344 by @avamingli
- #334 by @tuhaihe
- #323 by @avamingli

## Tools and more

We have recently released the backup utility, [gpbackup for CloudberryDB](https://github.com/cloudberrydb/gpbackup). We invite you to try it out and look forward to the upcoming tools and extensions for CloudberryDB.

Additionally, we have finished the prototype work for our new website, which will be developed in the coming months. Please stay tuned.

We also want to give a shoutout to our CloudberryDB documentation team who are working hard to verify and organize documents. Thank you so much for your dedication!

## Contributors
🎈️🎊️ Thanks to the following contributors for helping make Cloudberry Database better this month:

> @songdongxiaoa2, @gfphoenix78, @shmiwy, @avamingli, @Ray-Eldath, @jiaqizho, @tuhaihe, @oppenheimer01, @IdaLee666, @TomShawn

## Join us
The Cloudberry Database community welcomes everyone to contribute, regardless of their level of experience. We encourage all types of contributions, no matter how small. Our [contribution guide](https://cloudberrydb.org/contribute/how-to-contribute) is available to help you get started with the process.

In addition, we offer various channels for community members to discuss, seek help, provide feedback, and chat. You can find [support](https://cloudberrydb.org/support) here. Let us know if you have any questions or feedback - we're always here to help!

Join us and be part of our community!

## Past Newsletters

- #319 
- #274
- #225
- #181

---

[![Slack](https://img.shields.io/badge/Slack-6a32c9)](https://communityinviter.com/apps/cloudberrydb/welcome) [![Twitter](https://img.shields.io/twitter/follow/cloudberrydb)](https://twitter.com/cloudberrydb) [![WeChat](https://img.shields.io/badge/WeChat-eebc46)](https://cloudberrydb.org/community/wechat) [![Website](https://img.shields.io/badge/Website-bbec46)](https://cloudberrydb.org) [![Youtube](https://img.shields.io/badge/Youtube-gebc46)](https://youtube.com/@cloudberrydb) [![Discussions](https://img.shields.io/badge/Forum-gebc46)](https://github.com/orgs/cloudberrydb/discussions)

<sub>Want to try it out quickly? Follow our [Bootcamp program](https://github.com/cloudberrydb/bootcamp) to get started with the Cloudberry Database Sandbox.</sub>

**Comment by TomShawn** [comment]
Hooray!
